web/src/server/api/routers/generations/getAllQuery.ts

```diff
@@ -62,26 +64,13 @@ export const getAllQuery = protectedProjectProcedure
     `,
     );
 
-    const scores = await ctx.prisma.score.findMany({
```

Comment:
I once pulled this out for performance reasons. Lets check toghether whether this works.